wallets/rn_cli_wallet/package.json

```diff
@@ -15,47 +16,63 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
+    "@craftzdog/react-native-buffer": "6.0.5",
     "@json-rpc-tools/utils": "1.7.6",
+    "@notifee/react-native": "7.8.2",
     "@react-native-async-storage/async-storage": "1.21.0",
     "@react-native-clipboard/clipboard": "1.13.2",
     "@react-native-community/netinfo": "11.1.0",
-    "@react-navigation/bottom-tabs": "^6.5.11",
+    "@react-native-firebase/app": "18.8.0",
+    "@react-native-firebase/messaging": "18.8.0",
+    "@react-navigation/bottom-tabs": "6.5.11",
+    "@react-navigation/elements": "1.3.21",
     "@react-navigation/native": "6.1.9",
+    "@react-navigation/native-stack": "6.9.17",
     "@react-navigation/stack": "6.3.20",
     "@sentry/react-native": "5.17.0",
+    "@walletconnect/notify-client": "0.16.5",
+    "@walletconnect/notify-message-decrypter": "^0.1.0",
     "@walletconnect/react-native-compat": "2.11.0",
     "@walletconnect/web3wallet": "1.10.0",
+    "dayjs": "^1.11.10",
     "ethers": "5.7.2",
+    "lodash.clonedeep": "^4.5.0",
     "patch-package": "^8.0.0",
     "postinstall-postinstall": "2.1.0",
+    "expo": "^49.0.0",
     "react": "18.2.0",
-    "react-native": "0.73.2",
+    "react-hook-form": "^7.50.1",
+    "react-native": "0.73.4",
     "react-native-device-info": "10.12.0",
     "react-native-dialog": "9.3.0",
     "react-native-gesture-handler": "2.14.1",
     "react-native-get-random-values": "1.9.0",
     "react-native-modal": "13.0.1",
+    "react-native-pager-view": "^6.2.3",
```

Comment:
are you using this lib?

Reply:
Yes. I was using it in `SubscriptionsScreen` but now I've removed it since it has some problems in Android and I don't want to over engineering it. I've simply replaced with a custom basic tab implementation which just toggles screens. Please review `SubscriptionsScreen` again.

Reply:
Ok, i'll review it! But if you are not using it anymore, uninstall this lib 🙏  and run pod-install again